src/coreclr/pal/inc/pal.h

```diff
@@ -2453,6 +2601,9 @@ PALIMPORT BOOL PALAPI PAL_VirtualUnwindOutOfProc(CONTEXT *context, KNONVOLATILE_
 #define PAL_CS_NATIVE_DATA_SIZE 56
 #elif defined(__sun) && defined(__x86_64__)
 #define PAL_CS_NATIVE_DATA_SIZE 48
+#elif defined(__linux__) && defined(__loongarch64)
+////TODO for LOONGARCH64: should confirm !!!
```

Comment:
It would be good to confirm this before merging this PR.

Reply:
Thanks.
I have update it.